src/main.js

```diff
@@ -10,6 +11,12 @@ Vue.use(vueDebounce, {
   listenTo: 'input'
 });
 
+Sentry.init({
```

Comment:
you can add a conditional by checking if the environment variable has value, if it has it, it activates sentry, otherwise you deactivate it